app.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+//const endpoints = require('../endpoints.json')
+
+const { getAllTopics, getDocs } = require("./controllers/topics.controllers");
+
+
+const app = express();
+
+
+
+app.get("/api/topics", getAllTopics);
+
+app.get('/api', getDocs)
+
+app.use(( req, res, next)=> {
```

Comment:
There is no path here. This should be app.all - and there should be a wildcard endpoint passed in here. Your callback looks fine